frame_id:
  map: "map"
  base_link: "base_link"

data_collection_node:
  lidarCloudTopic: "/velodyne/points"
  globalMapPath: "$(find height_mapping)/../maps/globalmap.bag" # Path to pre-saved global map
  savePath: "$(find height_mapping)/../data/"
  dataCollectionPeriod: 1.0 # [s]
  publishRate: 10.0 # [Hz], for visualization

data_collection:
  # Height map parameters
  minHeightThreshold: -0.2 # [m]
  maxHeightThreshold: 1.5 # [m]
  mapLengthX: 15.0 # [m]
  mapLengthY: 15.0 # [m]
